README.md

```diff
@@ -6,14 +6,44 @@ This is a fork of the excellent [`fuser`](https://github.com/cberner/fuser) Rust
 
 This fork should be maintained in the `fuser/fork` branch of the [awslabs/mountpoint-s3 repository on GitHub](https://github.com/awslabs/mountpoint-s3/tree/main/vendor/fuser).
 
-To pull in new changes from upstream, you should fetch and rebase the changes locally and then force push to the `fuser/fork` branch (- not ideal!).
-Once the `fuser/fork` branch is as desired, create a new branch from Mountpoint's `main` branch.
-Run the [`vendor-fuser.sh` script](https://github.com/awslabs/mountpoint-s3/blob/main/vendor-fuser.sh).
-You should open a pull request with the new commit created by the script.
-This pull request is where the changes are reviewed.
-
-If you wish to add new divergent changes to this fork of Fuser,
-open a pull request on the Mountpoint repository branching from the `fuser/fork` branch and use `fuser/fork` as the base branch when creating the pull request.
+#### Sync with the origin
+
+To pull in new changes from upstream, you should fetch changes from the original upstream and rebase them locally.
+
+1. Add a new remote if haven't done it already `git remote add fuser https://github.com/cberner/fuser.git`
+2. Fetch from the remote `git fetch fuser`
+3. List all the FUSER's tags `git tag -l | grep v`, it'll output all of the FUSER versions like `v0.15.1`. Select the version that you'd want to sync with (latest one by default)
+4. **Important** Next step implies history change, so at this point we need to make sure that we have a snapshot of the current state of `fuser/fork` branch. It must be a tag, e.g. `fuser-fork-<version>` containing the head commit. We assume we have it since we supposed to be following this runbook during the previous sync.
+    1. Create a new branch from `fuser/fork`, `git rebase v0.15.1 -i` there and pick our commits which we'd like to put on top of the target tag (all of the commits by default)
+    2. Once rebase is finished and all the possible conflicts (at least in `Cargo.toml`) are solved bump the fork's version and commit the change. Original version is not equal to fork's version, but we have to bump either major or patch part of it depending on what's changed in the original
```

Comment:
I think we can tone down the explanation here if we cover it before the instructions. This can boil down to "we're going to now tag this commit so it can be referenced by submodules".

I think we can drop the assumption of the previous sync, it only adds complexity to the instructions here.